core/cmd/client.go

```diff
@@ -529,7 +529,7 @@ func confirmAction(c *clipkg.Context) bool {
 	prompt := NewTerminalPrompter()
 	var answer string
 	for {
-		answer = prompt.Prompt("Are you sure? This action is irreversible! (yes/no)")
+		answer = prompt.Prompt("Are you sure? This action is irreversible! (yes/no) ")
```

Comment:
Why add a space?

Reply:
So that when you begin typing `yes` or `no`, it's not jammed up against the `)`.